### PR TITLE
doc: add v2 ApisixRoute reference

### PR DESCRIPTION
**Pull request by stillfox-lee**
### Type of change:

- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

### What this PR does / why we need it:

[issue](https://github.com/apache/apisix-ingress-controller/issues/1063)

I just copy from api_route_v2beta3.md. And add some new rows that already exist in https://github.com/apache/apisix-ingress-controller/blob/d3a823f590843468a555ccc68502acc568d9a9f3/samples/deploy/crd/v1/ApisixRoute.yaml#L563.

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

### CODE REVIEW

**Comment by tao12345666333** [comment]
Can you help to add the v2 documentation for the other resources as well?

**Comment by stillfox-lee** [comment]
> Can you help to add the v2 documentation for the other resources as well?

Sure. Sorry for missing the other resources.

**Comment by tao12345666333** [comment]
Don't worry, ping me if you need any help

**Comment by stillfox-lee** [comment]
Hi @tao12345666333 , I have a question about our CRD version 2. 
In ApisixPluginConfig and ApisixUpstream CRD yaml. There are no v2 but only v2beta3. Maybe we should define v2 CRD then add v2 documentation? Otherwise, doc and our implementation are not unified.

**Comment by tao12345666333** [comment]
@stillfox-lee If you pull the latest code from the master branch, you will see the CRD configuration of ApisixPluginConfig v2.

For ApisixUpstream, it can be ignored because we haven't added the v2 implementation yet. until #944  merged

**Comment by stillfox-lee** [comment]
> @stillfox-lee If you pull the latest code from the master branch, you will see the CRD configuration of ApisixPluginConfig v2.
> 
> For ApisixUpstream, it can be ignored because we haven't added the v2 implementation yet. until #944 merged

Got it, thx.

**Comment by tao12345666333** [comment]
I can merge the commit records for you later, so that files unrelated to your changes will not appear in the review list

**Comment by stillfox-lee** [comment]
> I can merge the commit records for you later, so that files unrelated to your changes will not appear in the review list

Oh, I see. Sorry about that. I'm not familiar with PR workflow. I should just merge upstream to my origin and then commit to the origin branch.

**Comment by stillfox-lee** [comment]
> I can merge the commit records for you later, so that files unrelated to your changes will not appear in the review list

Oh, I see. Sorry about that. I'm not familiar with PR workflow. I should just merge upstream to my origin and then commit to the origin branch.

**Comment by tao12345666333** [comment]
Sorry for delay.

I have updated the commit records in this PR.

File: docs/en/latest/references/apisix_cluster_config_v2beta3.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: ApisixClusterConfig/v2alpha1 Reference
```
Comment (tao12345666333) [inline docs/en/latest/references/apisix_cluster_config_v2beta3.md:2]:
```suggestion
title: ApisixClusterConfig/v2beta3 Reference
```

File: docs/en/latest/references/apisix_route_v2.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+---
+title: ApisixRoute/v2beta3 Reference
```
Comment (tao12345666333) [inline docs/en/latest/references/apisix_route_v2.md:2]:
```suggestion
title: ApisixRoute/v2 Reference
```

File: docs/en/latest/references/apisix_tls_v2beta3.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+title: ApisixTls Reference
```
Comment (tao12345666333) [inline docs/en/latest/references/apisix_tls_v2beta3.md:2]:
```suggestion
title: ApisixTls/v2beta3 Reference
```

**Review by tao12345666333 — COMMENTED** [review]
The titles of these files also need to be updated, thanks!

**Review by tao12345666333 — APPROVED** [review]
LGTM

**Pull request merged**